### ISSUE TITLE
♿Date Picker AVT1 - August 2018♿

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#1331]
## Environment
Chrome Version 70.0.3538.67 (Official Build) (64-bit)
macOS Mojave 10/Windows 10

## Violations:
Each form must have an associated label
<img width="567" alt="screen shot 2018-10-25 at 4 06 22 pm" src="https://user-images.githubusercontent.com/40970507/47530344-f2b94e00-d86f-11e8-8122-efc41dcc68a2.png">

<img width="805" alt="screen shot 2018-10-25 at 3 44 15 pm" src="https://user-images.githubusercontent.com/40970507/47529946-e2ed3a00-d86e-11e8-94fb-bb6f001100dc.png">

Two input elements have the same id. Id's should be unique
<img width="575" alt="screen shot 2018-10-25 at 4 08 22 pm" src="https://user-images.githubusercontent.com/40970507/47530468-4deb4080-d870-11e8-92cc-c3f300210daa.png">

## How to reproduce
1. [Download the Dynamic Accessibility Plugin tool here](https://ibm.ent.box.com/notes/335005856253?s=mhpgjlgss354mcidsv6rvpuu0bxlmfm4)
2. Run the extension using the August 2018 ruleset

**Comment by dakahn** [comment]
Closed this out cause it was absent mindedly overwritten. :hurtrealbad:
